addons/mail/static/src/utils/common/hooks.js

```diff
@@ -89,6 +89,29 @@ export function useHover(refName, callback = () => {}) {
     return state;
 }
 
+/**
+ * Hook that execute the callback function when the scrollable element hit the
+ * bottom.
+ *
+ * @param {string} refName scrollable t-ref name to observe
+ * @param {function} callback function to execute when scroll hit the bottom
+ * @param {number} threshold number of threshold pixel to trigger the callback
+ */
+export function useOnBottomScrolled(refName, callback, threshold = 1) {
```

Comment:
name and comment are misleading, it is not executed when the bottom is reached (assumed only once), but also every time afterwards as long as the threshold is still applicable and there is a scroll event

Reply:
I changed the comment, I have no idea how to correctly name this function, threshold is an option of this hook so I think it's fine to keep the current one.
Or you have something else.